services/ui-src/src/types/index.ts

```diff
@@ -124,6 +124,37 @@ export interface TimeShape {
   second: number;
 }
 
+// Widgets
+export interface iconlessWidget {
+  type: string;
+  content: {
+    leftBarColor: string;
+    title: string;
+    descriptionList: string[];
+  };
+}
+
+export interface iconWidget {
```

Comment:
Maybe iconWidget could extend iconlessWidget, since they share the same properties?

Reply:
If I keep the same format extending looks like this:

```
export interface iconWidget extends iconlessWidget {
  content: {
    leftBarColor: string;
    icon: string;
    iconDescription: string;
    title: string;
    descriptionList: string[];
    additionalInfo?: string;
  };
}
```

just by way of the content object needing to be defined.. Could change up these types though so the content part is actually what the interface is and was something I was debating for quite awhile before settling on combining them. Thoughts on that?

Reply:
Hmm I see what you mean. It looks incomplete this way. Ok I'm cool with keeping them separate! Maybe Brax will have an opinion here.

Reply:
Could make a `WidgetContent` interface that includes most of that. Then extend it to an `IconWidgetContent` and `IconlessWidgetContent` (as needed) that could be extended in those other ones. Seems to me like you're only really ever accessing the content anyway, and even then it could probably just be declared as the `Props` in the actual widget component files. Happy to pair on this.